quill-core/src/main/scala/io/getquill/util/Messages.scala

```diff
@@ -4,6 +4,11 @@ import scala.reflect.macros.blackbox.{ Context => MacroContext }
 
 object Messages {
 
+  private val debugEnabled =
```

Comment:
I mean the problem is that it's really not usable as soon as you hit a certain amount of queries, also quill still produces runtime logs for the query, which is more than enough. (this is only compile time removal)